src/qibo/optimizers.py

```diff
@@ -35,11 +35,10 @@ def optimize(loss, initial_parameters, args=(), method='Powell',
         processes (int): number of processes when using the parallel BFGS method.
 
     Returns:
-        loss (float): final best loss value.
-        xbest (float): best parameters obtained by the optimizer.
-        extra: optimizer specific return object containing. For scipy methods it
-            returns the ``OptimizeResult``, for ``'cma'`` the ``CMAEvolutionStrategy.result``,
-            and for ``'sgd'`` the options used during the optimization.
+        (float, float, custom): Final best loss value; best parameters obtained by the optimizer;         extra: optimizer specific return object containing. For scipy methods it
```

Comment:
```suggestion
        (float, float, custom): Final best loss value; best parameters obtained by the optimizer;         extra: optimizer-specific return object. For scipy methods it
```
Let me know if you agree.